.github/workflows/ci-image-build.yml

```diff
@@ -189,7 +162,14 @@ ${{ inputs.do-build == 'true' && inputs.image-tag || '' }}"
           GITHUB_USERNAME: ${{ github.actor }}
           PUSH: ${{ inputs.push-image }}
           VERBOSE: "true"
-        if: inputs.do-build == 'true'
-      - name: "Stop ARM instance"
-        run: ./scripts/ci/images/ci_stop_arm_instance.sh
-        if: always() && inputs.do-build == 'true' && inputs.platform == 'linux/arm64'
+      - name: "Export CI docker image ${{ env.PYTHON_MAJOR_MINOR_VERSION }}"
+        run: breeze ci-image save --platform "${{ inputs.platform }}"
```

Comment:
```suggestion
        run: breeze ci-image save --platform "${{ inputs.platform }}" --python ${{ env.PYTHON_MAJOR_MINOR_VERSION }}"
```

Reply:
I prefer to pass it **just** via `env.PYTHON_MAJOR_MINOR_VERSION`. You never have to do `--python` when PYTHON_MAJOR_MINOR_VERSION env is set, because this is what envvar sets:

```
option_python = click.option(
    "-p",
    "--python",
    type=CacheableChoice(ALLOWED_PYTHON_MAJOR_MINOR_VERSIONS),
    default=CacheableDefault(value=ALLOWED_PYTHON_MAJOR_MINOR_VERSIONS[0]),
    show_default=True,
    help="Python major/minor version used in Airflow image for images.",
    envvar="PYTHON_MAJOR_MINOR_VERSION",
)
```